openedx/core/djangoapps/zendesk_proxy/v0/views.py

```diff
@@ -0,0 +1,57 @@
+"""
+Define request handlers used by the zendesk_proxy djangoapp
+"""
+from rest_framework import status
+from rest_framework.parsers import JSONParser
+from rest_framework.response import Response
+from rest_framework.throttling import UserRateThrottle
+from rest_framework.views import APIView
+
+from openedx.core.djangoapps.zendesk_proxy.utils import create_zendesk_ticket
+
+ZENDESK_REQUESTS_PER_HOUR = 15
```

Comment:
I'd still like to double check with legal on the implications of this rate; we'll be preventing anyone from making an a11y request at all if we hit this limit in a single hour.

Reply:
This has been confirmed okay

Reply:
It may be that the rates will be different for other applications (e.g. LMS vs. CMS) or even endpoints. I'm ok deferring the optimization for later, but wanted to raise it.

Reply:
That definitely sounds like a "v1 or later" type of enhancement :)